### ISSUE TITLE
Account manager:  US country code (+1) appearing in front of any phone number

### ISSUE THREAD

**Issue by SteeveEbener** [terraframe/geoprism-registry#162]
**List the CGR version information**
See posting date of the issue to get the version information 

**Describe the bug**
The US country code (+1) appears by default in front of any phone number you associate with a user account

**To Reproduce**
1. Go to the account manager
2. Look at your account information (if you are not based in the US you will already see the "+1" in front of your phone number
3. Click on edit, the +1 is not appearing in the field and can't therefore be removed

**Expected behavior**
The correct country code appears in front of the user phone number either automatically (country setting?) or manually 

**Screenshots**
![image](https://user-images.githubusercontent.com/12602163/78528216-71f6d380-7811-11ea-854a-064a407a342d.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version: 80.0.3987.163 (Official Build) (64-bit)

**Comment by CarlyNel** [comment]
@SteeveEbener @jsmethie  I just tested this - feedback as follows:

1) I'm logged in as MOH-RA
2) Navigated to Settings -> Manage accounts
3) Clicked the edit pen on the user Carly Test
4) I updated two numbers under my profile. #1: I included the SA dialling code of +27 For #2: I simply typed my cell number starting with 079. 
5) I saved my changes but the summary user accounts screen is still showing 1) The old number that was originally entered under this test profile and 2) with the +1 dialing code which I didn't enter. 
6) As a side note, I also changed the user name but that also hasn't updated on the summary page.
7) I then refreshed the page and only after doing so, did my changes reflect.

Screenshots below.

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/afade7b3-2cd7-4d4b-93c6-4ddbd59814ff)

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/cbe2aded-929d-4bfd-92ee-f207d4af17ea)

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/3a7b5171-a065-4838-9c47-3d7045902a93)

**Comment by jsmethie** [comment]
@CarlyNel Just to make sure I understand, everything is correct after you refresh the accounts page?  The issue now would be that the table doesn't reflect changes until the whole page is refreshed?

**Comment by CarlyNel** [comment]
Hi Justin @jsmethie, that's correct. However, my brower's cache just caused an issue with another ticket that I was testing so I cleared my cache and after doing so, I could see the correct information on the other ticket that I was expecting to see (and which is in staging). Perhaps my cache was causing this supposed 'refresh' issues in this instance too. I'm not sure whether it will make a difference but I'll test this again now and revert. Will also wait for Steeve's feedback once he's tested this on his side.

**Comment by CarlyNel** [comment]
Hi @jsmethie, Ok I've just tested this again and have experienced the same issue of the updated info not reflecting on the home screen for account management until I've refreshed the page. Screenshots as follows:

1) The info I just updated:

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/58081a3d-a46a-4e47-a954-4d8e7c060508)

2) The summary after I've submitted my changes:

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/1b3adb85-6152-43f9-bf3e-9025a5ff21bb)

3) The summary after I've refreshed the page:

![image.png](https://images.zenhubusercontent.com/5eda94a201596292f642f925/dec66ed9-e02f-4469-a8a0-47746c0981f4)

Hope this helps.

**Comment by jsmethie** [comment]
Great.  I have created a new ticket specific to the refreshing problem:

https://app.zenhub.com/workspaces/geoprism-registry-5df81d4b734ff217d40b12a4/issues/terraframe/geoprism-registry/355

It will be fixed in the next release.  Please close this ticket just based on the phone number behavior.

**Comment by SteeveEbener** [comment]
Thanks guys. As per the rule we decided I should be the one closing this ticket as I opened it. I will do that as soon as I have also tested this from my side. Thanks

**Comment by SteeveEbener** [comment]
Checked from my side as well. Closing the ticket.
